### ISSUE TITLE
min-height bug when changing from jquery 2.3.3 to jquery 3.1.1 

### ISSUE THREAD

**Issue by anonym3000** [almasaeed2010/AdminLTE#1204]
if you update jquery to version 3.1.1, the app.js that changes the min-weight of the "content-wrapper" class would not work

**Comment by Candanz** [comment]
In app.js change line 252 to:
var neg = $('.main-header').outerHeight() + $('.main-footer').outerHeight() || 0;

This fixxed it for me.
It seems that jQuery  2.3.3 returns "null" when reading the outer height of a non-existant element.
This was changed in jQuery 3, and it now returns NaN.

50 + null = 50
50 + NaN = NaN

So the min-height wasn't actually being set because the calculation results into NaN

**Comment by anonym3000** [comment]
Thanks! I removed $('.main-footer').outerHeight() and it works now.

**Comment by DiomedesDominguez** [comment]
Why there isn't an official version of AdminLTE using jQuery 3+?

**Comment by DiomedesDominguez** [comment]
For me, this worked in Edge, Firefox, IE11 and Chrome:

```javascript
      //Get window height and the wrapper height
      var neg = $('.main-header').outerHeight() + $('.main-footer').outerHeight() || 0;
      var window_height = $(window).height() ;
      var sidebar_height = 0;
      if ($(".sidebar").height() !== undefined) {
          sidebar_height = $(".sidebar").height();
      }
```
$('.main-footer').outerHeight() is returning 51, but  $(".sidebar").height() its returning undefined when used the Top Navigation configuration.

**Comment by chrislandeza** [comment]
What is the best way to fix this w/o changing the Core app.js?

**Comment by DiomedesDominguez** [comment]
That's the only way.

**Comment by almasaeed2010** [comment]
@chrislandeza fixed in #1276
